src/main/java/com/google/cloud/tools/appengine/cloudsdk/CloudSdkAppEngineDeployment.java

```diff
@@ -82,7 +82,7 @@ public void deploy(DeployConfiguration config) throws AppEngineException {
     arguments.addAll(GcloudArgs.get("server", config.getServer()));
     arguments.addAll(GcloudArgs.get("stop-previous-version", config.getStopPreviousVersion()));
     arguments.addAll(GcloudArgs.get("version", config.getVersion()));
-    arguments.addAll(GcloudArgs.get(config));
+    arguments.addAll(GcloudArgs.get("project", config.getProjectId()));
```

Comment:
this change looks good, but should it be pulled into a separate PR?

Reply:
Since deployment no longer extends the `Configuration` interface, this had to be made here.